overlord/devicestate/firstboot_test.go

```diff
@@ -759,7 +759,7 @@ func (s *firstBoot16Suite) TestPopulateFromSeedConfigureHappy(c *C) {
 defaults:
     foodidididididididididididididid:
        foo-cfg: foo.
-    coreidididididididididididididid:
+    99T7MUlRhtI3U0QFgl5mXXESAiSwt776:
```

Comment:
Silly question but why is this not using `naming.WellKnownSnapID("core")` ?

Reply:
it's inside a string, so it looks like what people will actually write in the world. We would need to %s or Replace this away, unclear it's a win

Reply:
one could add a comment inside though